### PR TITLE
gui: Introduces swupd config page

### PR DESCRIPTION
**Pull request by reaganlo**
**Please ensure the above *guidelines for contributing* are met.**

Fixes Issue: #405 #407 #455

Changes proposed in this pull request:
- Introduces swupd config page that can be used to set Mirror URL and Auto Updates
- Moved associated common GUI and TUI strings to swupd.go
- Added corresponding translations
- Made the change to display swupd related pages in sequence in the TUI menu.
- Fixed resize issue due to long strings in **User Add** page #455 as part of fixing the same for this page.
- Refactored code to handle closing pages due to a bug introduced in this PR.

### CODE REVIEW

File: swupd/swupd.go [hunk 1]
```diff
@@ -412,7 +459,7 @@ func setMirror(swupdArgs []string, t string) (string, error) {
 func SetHostMirror(url string) (string, error) {
 
 	if urlErr := network.CheckURL(url); urlErr != nil {
-		return "", fmt.Errorf("Server not responding")
+		return "", fmt.Errorf(utils.Locale.Get("Server not responding"))
```
Comment (mdhorn) [inline swupd/swupd.go]: Initially, I had concerns about having these translated in common code as this means the TUI will get translated strings that might now format right for other languages. But looking through the code base we have similar issues elsewhere, so we will not worry about this for now.

File: gui/pages/swupd_config.go [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+func (page *SwupdConfigPage) GetConfiguredValue() string {
+	var ret string
+	if page.model.AutoUpdate {
+		ret = utils.Locale.Get("Auto updates enabled")
+	} else {
+		ret = utils.Locale.Get("Auto updates disabled")
+	}
+
+	if page.model.SwupdMirror != "" {
+		ret += utils.Locale.Get(". Custom mirror set.")
```
Comment (mdhorn) [inline gui/pages/swupd_config.go]: I'm not sure that it makes sense to have this leading person here for the previous statement.

**Comment by reaganlo** [comment]
@mdhorn As discussed, I've made the following changes:
- Added corresponding translations
- Made the change to display swupd related pages in sequence in the TUI menu.
- Fixed resize issue due to long strings in **User Add** page #455 as part of fixing the same for this page.
- Refactored code to handle closing pages due to a bug introduced in this PR.

File: gui/pages/swupd_config.go (hunk 2 above)
Comment (reaganlo) [reply gui/pages/swupd_config.go]: Done to have "." as a separate translated string.

**Pull request merged**